package.json

```
@@ -41,74 +40,59 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@babel/node": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/preset-env": "^7.1.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^23.6.0",
-    "cross-env": "^5.2.0",
-    "es-check": "^4.0.0",
-    "eslint": "^5.7.0",
-    "eslint-config-jest-files": "^0.1.3",
-    "eslint-config-unicorn-camelcase": "^0.1.1",
-    "eslint-plugin-prettier": "^3.0.0",
-    "flux-standard-action": "^2.0.3",
-    "husky": "^1.1.2",
-    "jest": "^23.6.0",
-    "lint-staged": "^7.3.0",
-    "npm-run-all": "^4.1.3",
-    "prettier": "^1.14.3",
-    "rimraf": "^2.6.2",
-    "rollup": "^0.66.6",
-    "rollup-plugin-babel": "^4.0.3",
-    "rollup-plugin-commonjs": "^9.2.0",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-terser": "^3.0.0",
-    "xo": "^0.23.0"
+    "@vitest/coverage-c8": "^0.26.2",
```

Shouldn't this be the exact version too? I'm assuming pnpm is installing as exact versions.

```suggestion
    "@vitest/coverage-c8": "0.26.2",
```